Source/monster.h

```diff
@@ -25,8 +25,8 @@ namespace devilution {
 
 struct Missile;
 
-#define MAXMONSTERS 200
-#define MAX_LVLMTYPES 24
+constexpr const int MaxMonsters = 200;
+constexpr const int MaxLvlMTypes = 24;

```

Comment:
constexpr implies const, no need to repeat in these definitions.

Reply:
@aetn23 would you be willing to do a follow up to correct this :)

Reply:
Sure